appveyor.yml

```diff
@@ -1,14 +1,13 @@
-init:
-  - git config --global core.autocrlf true
-branches:
-  only:
-    - master
-    - release
-    - dev
-    - /^(.*\/)?ci-.*$/
+image: Windows Server 2016
```

Comment:
Should be updated to `Visual Studio 2017` to have official Docker build in AppVeyor.